### PR TITLE
Use putIfAbsent

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#806](https://github.com/nf-core/sarek/pull/806) - Refactor all tests
 - [#806](https://github.com/nf-core/sarek/pull/806) - Split up `modules.config` file
 - [#810](https://github.com/nf-core/sarek/pull/810) - Update CHANGELOG
```
Comment (maxulysse) [inline CHANGELOG.md:27]: What's happening here?

**Review by maxulysse — COMMENTED** [review]
Looks good. Had no idea about this option. Just some issues with changelog

File: CHANGELOG.md (hunk 1 above)
Comment (FriederikeHanssen) [reply CHANGELOG.md:27]: 🤨 was that you test run?

Comment (FriederikeHanssen) [reply CHANGELOG.md:27]: https://github.com/nf-core/sarek/pull/810 I think it was, can prob remove that line

Comment (maxulysse) [reply CHANGELOG.md:27]: Did we merge it?

Comment (FriederikeHanssen) [reply CHANGELOG.md:27]: yes it was merged in 
Screenshot 2022-11-10 at 00 21 49

**Review by maxulysse — APPROVED** [review]

**Pull request merged**